4MI0800264/Week 3/Problem 2.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+int findCharsDifferentFromAandD()
+{
+	int count = 0;
+	ifstream file("MyText.txt");
```

Comment:
Why's this file name hard-coded ? It should at least be an argument to the function :thinking: